src/appleseed/libappleseed.map

```diff
@@ -1,9 +1,4 @@
 {
-global: *;
-local:
-    extern "C++"
-    {
-        OSL*;
-        boost*;
-    };
+    global: *;
+    local: TIFF*; _TIFF*; _tiff*; png*; jpeg*;
```

Comment:
No longer hiding `OSL*` and `boost*`?

Reply:
By default we don't use this file in builds. We were not hidding (and still don't) any symbol.
Hiding boost and OSL was not working. They are not a problem in any case, their versions are fixed in the vfx platform.

I tested that with the changes here and enabling the USE_VISIBILITY_MAP cmake option 
symbols from C image IO libs are correctly hidden.

Reply:
Should `USE_VISIBILITY_MAP` be ON by default? Should the `experimental` status from the option's description be removed?

Reply:
I was planning to do it after testing a few days locally.